.pre-commit-config.yaml

```diff
@@ -19,7 +19,10 @@ repos:
     rev: "7.1.0"
     hooks:
       - id: flake8
-        args: [--config=.flake8]
+        args: [
+          --config=.flake8,
+          --extend-ignore=FCN001
```

Comment:
Why do you ignore this?

Reply:
due to this line:
https://github.com/RedHatQE/cloud-tools/blob/8edce8d8b3c8f7adaf226054126b1565a4a27a0f/clouds/tests/aws/test_aws_utils.py#L93

It throws keyword argument missing error

Reply:
please check https://github.com/RedHatQE/openshift-cli-installer/blob/main/.flake8 for example how to exclude

Reply:
done